integration-cli/docker_api_containers_test.go

```diff
@@ -232,7 +232,7 @@ func TestContainerApiStartVolumesFrom(t *testing.T) {
 // This is pretty much the same as TestRunApplyVolumesFromBeforeVolumes, except with passing the VolumesFrom and the bind on start
 func TestVolumesFromHasPriority(t *testing.T) {
 	defer deleteAllContainers()
-	volName := "voltst"
```

Comment:
It was always failing w conflict container exists because I think a race w
deleting containers from above test :(

On Wednesday, April 1, 2015, Tibor Vass <notifications@github.com> wrote:

> In integration-cli/docker_api_containers_test.go
> <https://github.com/docker/docker/pull/12012#discussion_r27627506>:
>
> > @@ -232,7 +232,7 @@ func TestContainerApiStartVolumesFrom(t *testing.T) {
> >  // This is pretty much the same as TestRunApplyVolumesFromBeforeVolumes, except with passing the VolumesFrom and the bind on start
> >  func TestVolumesFromHasPriority(t *testing.T) {
> >  	defer deleteAllContainers()
> > -	volName := "voltst"
>
> @jfrazelle <https://github.com/jfrazelle> huh??
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12012/files#r27627506>.
>